dm/loader/lightning.go

```diff
@@ -281,6 +281,10 @@ func (l *LightningLoader) restore(ctx context.Context) error {
 			return err
 		}
 		cfg := lcfg.NewConfig()
+		// TableConcurrency is adjusted to the value of RegionConcurrency
+		// when using TiDB backend.
+		// TODO: should we set the TableConcurrency separately.
+		cfg.App.RegionConcurrency = l.cfg.LoaderConfig.PoolSize
```

Comment:
seems put this after `cfg.LoadFromGlobal` is more similar to other logic

Reply:
[df5017c](https://github.com/pingcap/tiflow/pull/6328/commits/df5017c185bb829304899d16e8fabeebb2825c04)